### PR TITLE
Refactor templates

### LINKED ISSUE #2227: Create narrow container page base template
### Describe the change
Some pages still mix wide page and narrow page layout:

image

image

This should be fixed, preferably by adding 2 distinct base templates to extend.

### Motivation
Easier templates

### Current implementation
Every template does this same styling over and over again

### Suggested implementation
have 2 base templates to extend

### PR DESCRIPTION
**Pull request by JobDoesburg**
Closes #2227 

### Summary
Refactors templates to use small_page or simple_page. Note that templates with multiple sections are not migrated.

### How to test
1. Make sure that all HTML pages look good and all JS still works on them

### CODE REVIEW

**Comment by pingiun** [comment]
The photo's page now doesn't show any content

File: website/singlepages/templates/singlepages/become_active.html [hunk 1]
```diff
@@ -1,28 +1,29 @@
-{% extends "base.html" %}
+{% extends "simple_page.html" %}
```
Comment (pingiun) [inline website/singlepages/templates/singlepages/become_active.html:1]: I think this page looks better as small_page

**Comment by se-bastiaan** [comment]
uh

**Comment by se-bastiaan** [comment]
whot

File: website/education/templates/education/books.html [hunk 2]
```diff
@@ -5,28 +5,25 @@
...
-            <p>
-                {% blocktrans trimmed %}
-                    If you cannot find a certain book or you find another
-                    error, please contact the board at
-                    <a href="mailto:books@thalia.nu" rel="noopener" target="_blank">books@thalia.nu</a>.
-                {% endblocktrans %}
-            </p>
-        </div>
-    </section>
+{% block page_content %}
```
Comment (pingiun) [inline website/education/templates/education/books.html:11]: I think this page looks better as a small_page

File: website/payments/templates/payments/bankaccount_list.html [hunk 3]
```diff
@@ -1,143 +1,132 @@
-{% extends "base.html" %}
+{% extends "simple_page.html" %}
```
Comment (pingiun) [inline website/payments/templates/payments/bankaccount_list.html:1]: I think this page looks better as a small_page

**Pull request merged**